src/main/java/org/codehaus/mojo/gwt/shell/JavaCommand.java

```diff
@@ -335,8 +335,7 @@ public void execute()
         {
             if ( timeOut > 0 )
```

Comment:
Thanks for the tip, have removed the 'if' block completely.

Timeout now fails the build, showing the stacktrace.